data_model/models.py

```diff
@@ -0,0 +1,109 @@
+        max_length=100, null=True
+    )  # classification/verdict/found/score/malscore
+    # HybridAnalysisObservable (verdict), BasicMaliciousDetector (malicious),
+    # GoogleSafeBrowsing (malicious), Crowdsec (classifications),
+    # GreyNoise (classification), Cymru (found), Cuckoo (malscore),
+    # Intezer (verdict/sub_verdict), Triage (analysis.score),
+    # HybridAnalysisFileAnalyzer (classification_tags)
+    external_references = pg_fields.ArrayField(
+        models.URLField(), null=True
+    )  # link/external_references/permalink/domains
+    # Crowdsec (link), UrlHaus (external_references), BoxJs,
+    # Cuckoo (result_url/permalink), Intezer (link/analysis_url),
+    # MalwareBazaarFileAnalyzer (permalink/file_information.value), MwDB (permalink),
+    # StringsInfo (data), Triage (permalink), UnpacMe (permalink), XlmMacroDeobfuscator,
+    # Yara (report.list_el.url/rule_url), Yaraify (link),
+    # HybridAnalysisFileAnalyzer (domains),
+    # VirusTotalV3FileAnalyzer (data.relationships.contacted_urls/contacted_domains)
+    related_threats = pg_fields.ArrayField(
+        models.CharField(max_length=100), null=True
+    )  # threats/related_threats
+    # GoogleSafeBrowsing, QuarkEngineAPK (crimes.crime)
+    malware_family = models.CharField(
+        max_length=100, null=True
+    )  # family/family_name/malware_family
+    # HybridAnalysisObservable, Intezer (family_name), Cuckoo, MwDB,
+    # Triage (analysis.family), UnpacMe (results.malware_id.malware_family),
+    # VirusTotalV3FileAnalyzer
+    # (attributes.last_analysis_results.list_el.results/attributes.names)
+    additional_info = (
+        models.JSONField()
+    )  # field for additional information related to a specific analyzer
+
+
+class DomainDataModel(BaseDataModel):
+    ietf_report = models.ForeignKey(
+        IETFReport, on_delete=models.CASCADE, null=True
+    )  # pdns
+    rank = models.IntegerField(null=True)  # Tranco
+
+
+class IPDataModel(BaseDataModel):
+    ietf_report = models.ForeignKey(
+        IETFReport, on_delete=models.CASCADE, null=True
+    )  # pdns
+    asn = models.IntegerField(null=True)  # BGPRanking, MaxMind
+    asn_rank = models.DecimalField(null=True)  # BGPRanking
+    certificates = models.JSONField(null=True)  # CIRCL_PSSL
+    org_name = models.CharField(max_length=100, null=True)  # GreyNoise
+    country = models.CharField(max_length=100, null=True)  # MaxMind, AbuseIPDB
+    country_code = models.CharField(max_length=100, null=True)  # MaxMind, AbuseIPDB
+    registered_country = models.CharField(
+        max_length=100, null=True
+    )  # MaxMind, AbuseIPDB
+    registered_country_code = models.CharField(
+        max_length=100, null=True
+    )  # MaxMind, AbuseIPDB
+    isp = models.CharField(max_length=100, null=True)  # AbuseIPDB
+    is_anonymizer = models.BooleanField(null=True)  # TorProject, Crowdsec
+    is_tor_exit_node = models.BooleanField(null=True)  # TorProject, Crowdsec
```

Comment:
I would argue that these are `tags`.

Maybe we could create an enum like this, that people could use for prearranged tags
```python3
file: enums.py

class DataModelTags(Choices):
     PHISHING = "Phishing"
     ANONYMIZER = "Anonymizer"
     TOR_EXIT_NODE = "TorExitNode"
     ...

file models.py

class BaseDataModel(models.Model):
    ...
    TAGS = DataModelTags
```

Reply:
👍🏻 yep it makes sense to use tags. We could also add the other tags available in Quokka right now (is_*)